### DISCUSSION TITLE
Mobile App: missing repo link in the release view

### DISCUSSION THREAD

**Discussion by jodydonetti** [community/community#3929] [Mobile] [answered]
Hi,
in the mobile app (I'm on iOS) when tapping on a notification for an update on an issue I'm taken to the issue view which contains (at the top) a link to quickly go to the related repo, see here:

![IMG_3924](https://user-images.githubusercontent.com/1010086/117710927-f402ba00-b1d2-11eb-97de-6b05c0566807.jpg)

When I do the same but for a notification about a new release, the release view does not contain that useful link, see here:

![IMG_3925](https://user-images.githubusercontent.com/1010086/117711054-10065b80-b1d3-11eb-8d13-d4ca7342a6ba.jpg)

Would it be possible to have that link there, too?

In my particular case I use that to change the watching settings for a repo I receive a notifications for, but in general I think it would also be more consistent between different (but similar) views.

Thanks!

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for the request! I sent your note to our team internally. Please keep the feature request and/or bug reports coming. We really appreciate it.

**Reply by jodydonetti** [reply]
Hi @elisealix22 , just checking: are there any news on this?

**Reply by elisealix22** [reply]
Amazing timing. We actually are shipping the updated release header to beta on Android today (v1.27), but it's still a work in progress on iOS! Should be coming soon. Thank you for following up and for pointing this out in the first place.

**Reply by jodydonetti** [reply]
Awesome, thanks!

**Reply by jodydonetti** [reply]
It's here on iOS, too 🎉

![image](https://user-images.githubusercontent.com/1010086/130876726-72b7e61e-a678-4e46-8a53-5f6aa07add41.jpeg)

Thanks!

**Comment by kytrun** [comment]
Hi, @elisealix22 !

Now on Android it can't be clicked, maybe it's a bug?

![0ae2e4e3b233c61aadf394542a1727e](https://user-images.githubusercontent.com/32519285/158332958-fedffa58-2862-43bf-8e0b-9aad29828f94.jpg)
